plugins/source/oracledb/client/types.go

```diff
@@ -3,59 +3,50 @@ package client
 import (
 	"strings"
 
-	"github.com/cloudquery/plugin-sdk/v2/schema"
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/cloudquery/plugin-sdk/v3/types"
 )
 
-func SQLType(t schema.ValueType) string {
-	switch t {
-	case schema.TypeBool:
+func SQLType(t arrow.DataType) string {
+	switch t.(type) {
+	case *arrow.BooleanType:
 		return "char(1)"
-	case schema.TypeInt:
+	case *arrow.Int8Type, *arrow.Int16Type, *arrow.Int32Type, *arrow.Int64Type,
+		*arrow.Uint8Type, *arrow.Uint16Type, *arrow.Uint32Type, *arrow.Uint64Type:
 		return "NUMBER(19)"
-	case schema.TypeFloat:
+	case *arrow.Float16Type, *arrow.Float32Type, *arrow.Float64Type:
 		return "binary_double"
-	case schema.TypeUUID:
+	case *types.UUIDType:
 		return "raw(16)"
-	case schema.TypeByteArray:
+	case *arrow.BinaryType, *arrow.LargeBinaryType:
 		return "blob"
-	case schema.TypeTimestamp:
+	case *arrow.TimestampType:
 		return "timestamp"
-	case schema.TypeJSON,
-		schema.TypeString,
-		schema.TypeStringArray,
-		schema.TypeUUIDArray,
-		schema.TypeCIDRArray,
-		schema.TypeMacAddrArray,
-		schema.TypeInetArray,
-		schema.TypeIntArray,
-		schema.TypeCIDR,
-		schema.TypeMacAddr,
-		schema.TypeInet:
-		return "clob"
 	default:
-		panic("unknown type " + t.String())
+		return "clob"
 	}
 }
 
-func SchemaType(tableName string, columnName string, dataType string) schema.ValueType {
-	if strings.HasPrefix(columnName, "timestamp") {
```

Comment:
There was a bug here actually. We should check `dataType` and not `columnName`.

Also we can remove the `tableName` and `columnName` arguments